### ISSUE TITLE
Add "KRE for start-ups" allocation

### ISSUE THREAD

**Issue by odednoam** [kinecosystem/rewards-engine#12]
One of the recurring concerns with regards to the KRE is that its incentives are distributed in proportion to a digital service's market size, which works well for established services but may be less efficient for small start-ups that don't have a lot of users. A lot of people (most people _I_ talked to) say we want both the established companies and the growing ones.

The trivial approach to incentivising start-ups in their growth phase is to allocate the rewards in proportion to _growth in market size_, rather than by _absolute market size_. 

For example, instead of distributing the daily rewards in proportion to the sum of stake of all wallets associated with an app, we could distribute the daily rewards in proportion to the difference in this sum of stake between now and 30 days ago. 

As discussed in my [Medium post](https://medium.com/kinfoundation/initial-feedback-on-the-kin-rewards-engine-5e91cad394aa), this method can easily be cheated, for example by zig-zagging users between apps, and other types of attacks. 

A way to mitigate these attacks would be to apply some kind of real-world accreditation to start-ups wanting to participate in the "growth" fund. For example, we thought of using real-world VCs  as a filter: known VCs would have the authority to register their portfolio start-ups to this program. In theory, one can still abuse the system in conspiracy with the VC, but such manipulations will be easy to spot. 

Few questions remain to be discussed:
1. What should be the allocation for the "start-ups" rewards? Is it a fixed % of the rewards or determined dynamically by the number of participants?
2. What institutes can be used to do the accreditation? Could it be VCs - and if so, how do we manage global and valid lists of VCs we can trust?
3. What should be the rules for start-ups? For example, should their participation be time-limited? Can they leave the program (and switch to the regular KRE) at any time? etc.

**Comment by ryana** [comment]
Happy to have some discussions going on here again!

I have two initial reactions:

First, the concern for startups feels way too heavy handed. What I mean by that is I don't see why we should be incentivizing startups to use the KRE in this way (and btw this is coming from someone who will almost certainly integrate KIN with at least 2 startups within the next 2 years). The bigger concern here is that without large retailers using KIN and providing pricing support for KIN in the exchanges, the KIN given to startups will be pretty low value so I'm wary to pre-optimize for this, disincentive larger companies, and create a less healthy marketplace in the process. I'd lean towards a more free-market approach.

Second, and more importantly, I find it hard to comment on this because I don't have a general sense for where this fits into the overall development of the KRE. To be blunt: I wonder if this is where anyone should be spending their time. The reason any of this is a problem to begin with is that KIN in it's current form doesn't have transaction fees, and while I'm no economist I've enough to have a pretty firm opinion that transaction fees are a necessary friction required to create working marketplaces. If future ERCs or something inherit to Stellar allow KIN to be transacted on Ethereum/Stellar with built-in fees, then I think a lot of the gaming attacks get washed out.

Ok now that I'm done there, to try to answer your question: I think using some sort of centralized approval process is fairly antithetical to crypto and should be avoided. If it can't be avoided, I think we should stay away from VCs. My spidy sense says that VCs in their current form will go the way of the dinosaur in the coming decades, and that transition will be accelerated among small businesses (e.g. most of my colleagues who have raised VC in prior businesses are now eschewing them at the early stages completely).

One very simple way to incentive smaller businesses would be to use a graduated bracket system like the US Federal tax code. Smaller businesses could simply get a higher % based on their volume. This would give smaller businesses a decent floor to operate and grow from.

**Comment by odednoam** [comment]
@ryana good points. I'll try to give _my_ opinion on what you raise but I totally agree that we should be discussing this to see what should be done. 

> I don't see why we should be incentivizing startups to use the KRE in this way. 

For me the importance of incentivising start-ups is to make the kin ecosystem the birthplace of the future giants. Using Kin makes a lot of sense to large apps today, but not necessarily to the largest (though it would be nice if it happened, none of us expect Facebook to join the Kin ecosystem). To help the ecosystem gain dominance I want it to be where new apps grow. In a way, it should work like Apple are promoting ("featuring") apps that were recently published and are growing fast.

> The bigger concern here is that without large retailers using KIN and providing pricing support for KIN in the exchanges, the KIN given to startups will be pretty low value so I'm wary to pre-optimize for this, disincentive larger companies, and create a less healthy marketplace in the process.

I don't think it's either-or. For Kin to win we need the ecosystem to enjoy a significant network effect, that will come from having enough earn & spend opportunities so that users don't go through exchanges (not that there's anything wrong with having an option to go to an exchange, I just don't think it's good for most users). As there are more earn and spend opportunities, the value to each app joining the ecosystem is bigger, creating a virtuous circle in which more apps will join and create more such opportunities.

> The reason any of this is a problem to begin with is that KIN in it's current form doesn't have transaction fees, and while I'm no economist I've read enough to have a pretty firm opinion that transaction fees are a necessary friction required to create working marketplaces. 

Why is it needed for working marketplaces (economically)? AFAIK there's no such need but will look into it, and consult our economic advisors. If you know of any evidence in that direction, I would appreciate sharing it.

That said, I agree that friction is very useful in mitigating many types of attacks. The problem is that fixed fees are destructive to micro-transactions, and variable fees are not supported by current blockchain platforms. Future platforms (including [Orbs](orbs.com) which is working closely with Kin, and in full disclosure, I'd remind you that I'm also on the Orbs team) will support variable fees and I definitely think we should make use of it, as I commented in issue #7 .

>  I think using some sort of centralized approval process is fairly antithetical to crypto and should be avoided.

I tend to agree on the principle, but so far we couldn't think of a way to prevent gaming of the system if we incentivise growth without such filter. IMHO defining a clear and transparent accreditation process would prevent favoritism or protectionism, and distributing only a small percentage of the rewards this way helps the process remain mostly decentralized. But I agree it's very delicate, and that if we over-do it it will be damaging. 

> If it can't be avoided, I think we should stay away from VCs

Can you think of some other institution we can use? I thought of VCs because I know (from my previous life) that cloud providers use VC and accelerators' accreditation as a filter in giving away start-up credits to their services.

> One very simple way to incentive smaller businesses would be to use a graduated bracket system like the US Federal tax code. 

But then, how do you prevent gaming of the system by splitting an app's users between different identities so that each one remains in the lowest bracket?

Thanks for commenting

**Comment by ryana** [comment]
In another thread I mentioned how hard it is to so solely use Github for discussions like this since you end up in "Quoting Hell" and this thread is already gonna be there I think haha :). Still, I'll do my best:

> For me the importance of incentivising start-ups is to make the kin ecosystem the birthplace of the future giants

KIN is already attractive. The KRE itself is attractive. The idea of using inflation to incentivize commerce is awesome. Nobody else is doing this. I just really don't think you have to overthink it _especially_ to incentivize businesses where 90% of them will fail in the 1st two years. KISS.

---

> Why is it needed for working marketplaces (economically)?

This topic is full of landmines but for a start, good 'ole Vitalik has a nice primer on this stuff: https://blog.ethereum.org/2016/07/27/inflation-transaction-fees-cryptocurrency-monetary-policy/

---

> Can you think of some other institution we can use?

No but remember I think that using any institution is a mistake :) I can think of a dozen businesses that do over $100k/month in revenue and never raised a dime of investor money (no VC, no accelerator) and would be a perfect fit for KIN. It's wrong to leave them out in the cold.

---

> How do you prevent gaming of the system by splitting an app's users between different identities

I don't think this is as much of an issue as it appears because, in practice, a big company has to do reporting and is tied to bank accounts and legal entities and tax considerations. I think that those realities make trying to game the system more trouble than it's worth. Even if an entity did try this, all that revenue/commerce eventually would have to consolidate into the main account, and you'd probably end up in a double taxation situation making this economically infeasible.

Furthermore, there's probably going to be some semblance of reputation somewhere right? I go to amazon.com and I buy things. There are probably dozens of different C corps and LLCs that I deal with when I transact with amazon.com, but in the end it all funnels up to Amazon.com, and the reputation of the address (amazon.com) is really valuable. Could amazon get lower taxes by having 300,000 separate websites all doing less revenue, maybe, but it's a hell of a lot more work for them, and those 300,000 separate websites are not going to have the same reputation of amazon.com, which adds more weight to the economic infeasibility argument.

So to summarize: with appropriately applied economic friction, you can make bad behavior simply cost too much. (This is related to the Vitalik link above.)

**Comment by bbrophy23** [comment]
Hi,

Big fan of the Kin project and participated in the TDE.

I have been thinking about the KRE and I believe I thought of a potential way to improve upon some of the ideas that have been put forth thus far.  Very simply, thus far as I have seen, the main idea has been to tie rewards to developers from KRE to in-app spending of Kin.  I think this needs to be flipped to more of a focus on in-app earning of Kin (though excluding KRE earnings themselves) for a number of reasons detailed below.

In many if not most cases, spending Kin in a single transaction will equal earning in that transaction (when one user spends, another user by default has to earn) so by linking the KRE to earning, the KRE would still incentive spending of Kin.  However, there are examples where earning Kin in a transaction won't equal spending and I think the Kin ecosystem will want to encourage these types of activities too.  

For example, if a developer wanted to make a Kin-based form of the Google search engine, there would be no opportunities to spend Kin but there would be a very big opportunity to earn Kin as ad revenue from search could be used to buy Kin and then distributed to users.  If the KRE is only linked to spending, something like this would never get built for the Kin ecosystem.  

Another example would be any digital good or service where the user does not have enough Kin to buy the digital good or service but does have enough fiat currency (which would essentially be for anyone using a Kin-based app for the first time).  A potential example would be a Kin-based form of Uber where a rider could pay with fiat, that fiat is converted to Kin and then rewarded to the driver without the 25% "tax" that Uber takes from the transaction.  Obviously, one could buy Kin off an exchange and/or earn enough Kin to pay for the Uber ride but in certain situations this would cause unnecessary friction for the user.

Expanding on this further, there are likely many users who simply either won't want to spend the time or won't easily earn enough Kin to buy all the digital goods and services they want in the Kin ecosystem and therefore will have to use fiat in some way.  Obviously, one could buy Kin off an exchange to pay for the digital good/service but in most situations this would cause unnecessary friction for the user.  By shifting the KRE to incentivize earning Kin, an app can still except fiat and participate in the KRE.

The main point is, by shifting the KRE from tied to spending to tied to earning, the Kin foundation would still encourage the spending of Kin but it would also open up an entire class of other use cases for Kin-based digital products that I believe the Kin ecosystem should be striving to support.

**Comment by odednoam** [comment]
@ryana regarding Vitalik's post on https://blog.ethereum.org/2016/07/27/inflation-transaction-fees-cryptocurrency-monetary-policy/ , Vitalik does not refer to any advantage to the health of the economy of having fees.

That said, when there are actual transaction costs, funding them through inflation rather than through fees is a state in which transaction issuers are not seeing the direct cost of their actions and that is expected to lead to inefficiencies (and there's a lot of economic literature on that subject, if anyone's interested, I recommend starting with Wikipedia articles [[1]](https://en.wikipedia.org/wiki/Free-rider_problem) [[2]](https://en.wikipedia.org/wiki/Subsidy#Economic_effects)). This is not the case at hand, because we're not talking about infrastructure transaction fees. 

Regarding the institutions problem, I think there will be no perfect solution to all types of start-ups. I too know start-ups that went through VCs and accelerators and ones that didn't (in the last few years I founded both a VC-backed start-up and a bootstrapped start-up). But _if_ we want a start-ups program in KRE, we need some institution that can reliably mark the eligible start-ups, and so far we have no suggestions other than VCs and accelerators. I guess bootstrapped start-ups will have to suffice with the regular plan, or apply through a VC if we allow the latter to process applications for companies that are not in their portfolio.

@bbrophy23 if I understand correctly, you're concerned that KRE applies to Kin spending but not to Kin earning. This is not the intention, but rather to reward both earners and spenders. I believe the RFC document reflects this approach, if you saw otherwise, please LMK and I'll correct the error.
